src/style.css

```diff
@@ -2,2 +1,82 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  /* Space between logo and nav */
+  align-items: center;
+  /* Vertically center the content */
+  padding: 1rem;
+  background-color: #fffff;
+  color: white;
```

Comment:
The text color is set to white, but the background-color is also very light, which will make the text unreadable. Ensure sufficient contrast between text and background colors.